packages/server/src/core/internals/config.ts

```diff
@@ -5,8 +5,8 @@ import { ErrorFormatter } from '../../error/formatter';
  * @internal
  */
 export interface RootConfigTypes {
-  ctx: Record<string, unknown>;
-  meta: Record<string, unknown>;
+  ctx: Record<any, unknown>;
+  meta: Record<any, unknown>;
```

Comment:
@sachinraja @iduuck  `{}` & `object` seemed to be valid base types too 🤔  - WDYT is best?